### PR TITLE
docs(apigateway): change "full working example" link to embed example

### PR DESCRIPTION
**Pull request by ghost**
The relative link given for the "full working example" works fine here on GitHub, but not when this is compiled into one of the API references.

Change the link to be a complete URL so it works from anywhere this content ends up.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-apigateway/README.md
Comment (shivlaks) [inline packages/@aws-cdk/aws-apigateway/README.md:568]: wondering if it's better to just include the example as this content is not really going to apply to Python, Java, .NET users but the README is published to corresponding package managers.

will defer to @nija-at though as he's the primary maintainer of this area :)

Comment (nija-at) [inline packages/@aws-cdk/aws-apigateway/README.md:568]: We should switch to a `.lit.ts` format so that this gets expanded and translated when published to package managers.

Example: 
source: https://github.com/aws/aws-cdk/blob/3800830609e67d6ffc4e18dd5d61bb67d90a066b/packages/%40aws-cdk/aws-servicediscovery/README.md#private-dns-namespace-example
rendered on api ref: https://docs.aws.amazon.com/cdk/api/latest/docs/aws-servicediscovery-readme.html#private-dns-namespace-example
rendered on python ref: https://docs.aws.amazon.com/cdk/api/latest/python/aws_cdk.aws_servicediscovery.README.html#private-dns-namespace-example

**Review by nija-at — CHANGES_REQUESTED** [review]

File: packages/@aws-cdk/aws-apigateway/README.md
Comment (Jerry-AWS) [reply packages/@aws-cdk/aws-apigateway/README.md:568]: I see. (Also see I linked the wrong example.)

**Comment by Jerry-AWS** [comment]
@nija-at Ready for another look.

File: packages/@aws-cdk/aws-apigateway/README.md
Comment (nija-at) [inline packages/@aws-cdk/aws-apigateway/README.md:529]: You will also need to change the name of the two files to have the extension `.lit.ts`.

**Review by nija-at — CHANGES_REQUESTED** [review]

**Comment by Jerry-AWS** [comment]
@nija-at OK, think I've got it now.

**Review by nija-at — APPROVED** [review]

**Review by shivlaks — APPROVED** [review]

**Review by shivlaks — CHANGES_REQUESTED** [review]
@Jerry-AWS - since the `integ` file itself was renamed, the `expected` files no longer match up.

**Comment by Jerry-AWS** [comment]
@shivlaks I think I fixed it but I see the CodeBuild is still failing... but I don't have permission to see the details

**Comment by nija-at** [comment]
@Jerry-AWS - a link to the build logs is posted as a comment. You can find it just above your previous comment.

The PR had to be merged from live and should fix the failing build.

**Pull request merged**
